datasets/laroseda/laroseda.py

```diff
@@ -0,0 +1,136 @@
+# coding=utf-8
+# Copyright 2021 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""LaRoSeDa: A Large Romanian Sentiment Data Set"""
+
+from __future__ import absolute_import, division, print_function
+
+import json
+
+import datasets
+
+
+# Find for instance the citation on arxiv or on the dataset repo/website
+_CITATION = """\
+@article{
+    tache2101clustering,
+    title={Clustering Word Embeddings with Self-Organizing Maps. Application on LaRoSeDa -- A Large Romanian Sentiment Data Set},
+    author={Anca Maria Tache and Mihaela Gaman and Radu Tudor Ionescu},
+    journal={ArXiv},
+    year = {2021}
+}
+"""
+
+# You can copy an official description
+_DESCRIPTION = """\
+        LaRoSeDa (A Large Romanian Sentiment Data Set) contains 15,000 reviews written in Romanian, of which 7,500 are positive and 7,500 negative.
+        Star ratings of 1 and 2 and of 4 and 5 are provided for negative and positive reviews respectively.
+        The current dataset uses star rating as the label for multi-class clasification.
+"""
+
+_HOMEPAGE = "https://github.com/ancatache/LaRoSeDa"
+
+_LICENSE = "CC BY-SA 4.0 License"
+
+# The HuggingFace dataset library don't host the datasets but only point to the original files
+# This can be an arbitrary nested dict/list of URLs (see below in `_split_generators` method)
+_URL = "https://raw.githubusercontent.com/ancatache/LaRoSeDa/main/data_splitted/"
+
+_TRAIN_FILE = "laroseda_train.json"
+_TEST_FILE = "laroseda_test.json"
+
+
+class LaRoSeDaConfig(datasets.BuilderConfig):
+    """BuilderConfig for the LaRoSeDa dataset"""
+
+    def __init__(self, **kwargs):
+        super(LaRoSeDaConfig, self).__init__(**kwargs)
+
+
+class LaRoSeDa(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class Laroseda(datasets.GeneratorBasedBuilder):
```
This must be the camel case version of the dataset script name `laroseda`
You'll need to regenerate the dataset_infos.json file after this change